plot_pixel_values.py

```diff
@@ -22,24 +24,70 @@ def get_opt():
     parser.add_argument('--fit-scaling',
                         action='store_true',
                         help='Fit the dark and scale factor (default=False)')
-
+    parser.add_argument('--plot-fits',
+                        action='store_true')
     parser.add_argument('--n-brightest',
-                        default=128,
+                        default=64,
                         type=int,
-                        help='Plot the N brightest (either 128 or n**2)')
+                        help='Plot the N brightest (either 64 or n**2)')
 
     args = parser.parse_args()
     return args
 
 opt = get_opt()
 
+
+def scale_model(pars, t_ccd):
+    scale, y_const, x_const = pars
+    return 1 / np.exp(np.log(scale) / 4.0 * (t_ccd - x_const))
+
+
+def fit_pix_values(t_ccd, esec):
+    logger = logging.getLogger("sherpa")
+    logger.setLevel(logging.WARN)
+    data_id = 1
+    ui.set_method('simplex')
+    ui.set_stat('cash')
+    ui.load_user_model(scale_model, 'model')
+    ui.add_user_pars('model', ['scale', 'y_const', 'x_const'])
+    ui.set_model(data_id, 'model')
+    ui.load_arrays(data_id,
+                   np.array(t_ccd),
+                   np.array(esec)
+                   )
```

Comment:
I didn't immediately see the best way to center and scale these data for stability, so I left constants in the "scale_model" .  To be fixed.